src/components/LogoItem.jsx

```diff
@@ -1,26 +1,22 @@
 import React from "react";
 
-const IconItem = ({
+const LogoItem = ({
   styles = "",
   iconStyles = "",
   titleStyles = "",
-  titleStyles2 = "",
-  src = "",
+  svgPath = "",
```

Comment:
Are you sure that you will get only svg icon? Use generic name like iconPath, iconUrl etc